example/scalalib/web/10-todo-webapp-cask-scalasql/build.mill

```diff
@@ -0,0 +1,49 @@
+package build
+
+import mill._, scalalib._
+
+object `package` extends RootModule with ScalaModule {
+  def scalaVersion = "2.13.8"
+
+  override def ivyDeps = Agg(
+    ivy"com.lihaoyi::cask:0.9.1",
+    ivy"com.lihaoyi::scalatags:0.13.1",
+    ivy"com.lihaoyi::upickle:3.1.0",
+    ivy"com.lihaoyi::scalasql:0.1.19",
+    ivy"com.h2database:h2:2.2.224"
+  )
+
+  object test extends ScalaTests {
```

Comment:
Let's have one `test` suite using H2 in memory and one `integration` suite using PostgreSQL via testcontainers.